pyproject.toml

```diff
@@ -96,10 +96,9 @@ docs = [
     "astropy[recommended]",  # installs the [recommended] dependencies
     "sphinx",
     "sphinx-astropy[confv2]>=1.9.1",
-    "pytest>=7.0",
```

Comment:
Why do we even have pytest as "docs" dependency?!

Reply:
I dug up https://github.com/astropy/astropy/pull/8974 for this. So should just leave it be, I guess.